src/utils/daos_hdlr.c

```diff
@@ -2239,9 +2240,11 @@ fs_copy(struct file_dfs *src_file_dfs,
 	int dfs_prefix_len,
 	const char *fs_dst_prefix)
 {
-	int rc = 0;
-	DIR *src_dir;
-	struct stat st_dir_name;
+	int	rc = 0;
+	DIR	*src_dir = NULL;
+	struct	stat st_dir_name;
+	char	filename[MAX_FILENAME];
+	char	dst_filename[MAX_FILENAME];
```

Comment:
Alignment.

Reply:
@ashleypittman I'm not seeing this alignment issue here?